### PR TITLE
E2e tests to check whether cluster autoscaling scale down works when one node pool is not autoscaled

### PR DESCRIPTION
**Pull request by mwielgus**
Follow up of #1301. 
cc: @piosz @jszczepkowski @fgrzadkowski

### CODE REVIEW

File: test/e2e/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -156,7 +156,7 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Slow]", func() {
 
 		By("Creating new node-pool with one n1-standard-4 machine")
 		const extraPoolName = "extra-pool"
-		addNodePool(extraPoolName, "n1-standard-4")
+		addNodePool(extraPoolName, "n1-standard-4", 1)
```
Comment (piosz) [inline test/e2e/cluster_size_autoscaling.go]: Do we have enough quota for such big machines?

Comment (mwielgus) [inline test/e2e/cluster_size_autoscaling.go]: Apparently yes as this test was passing for the last couple weeks.

**Comment by piosz** [comment]
LGTM

**Comment by piosz** [comment]
Merging manually to let the test pass few times on Jenkins.

cc @fgrzadkowski

**Pull request merged**